### PR TITLE
Add quick_patch module and update game_console module

### PR DESCRIPTION
**Pull request by Joelrau**

### CODE REVIEW

File: src/client/module/quick_patch.cpp [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+#include "module/game_console.hpp"
+
+quick_patch::quick_patch()
+{
+
+}
+
+void quick_patch::post_unpack()
+{
+	quick_patch::initialize();
```
Comment (momo5502) [inline src/client/module/quick_patch.cpp:14]: Why an initialize method? just put your stuff in here

File: src/client/module/quick_patch.cpp [hunk 2]
```diff
@@ -0,0 +1,81 @@
+#include <std_include.hpp>
+#include "quick_patch.hpp"
+#include "game/game.hpp"
+#include "module/command.hpp"
+#include "module/game_console.hpp"
+
+quick_patch::quick_patch()
```
Comment (momo5502) [inline src/client/module/quick_patch.cpp:7]: Jimbo already started working on a patches module which is essentially the same. Maybe you should coordinate with him, to not cause merge conflicts

File: src/client/module/quick_patch.cpp [hunk 3]
```diff
@@ -0,0 +1,81 @@
...
+	// patch cg_fov
+	game::native::Dvar_RegisterFloat("cg_fov", 65.0f, 65.0f, 120.0f, 0x1, "The field of view angle in degrees"); // changed max value from 80.0f -> 120.f
+
+	// patch r_vsync
+	game::native::Dvar_RegisterBool("r_vsync", false, 0x1, "Enable v-sync before drawing the next frame to avoid 'tearing' artifacts."); // changed default value from true -> false. There are some fps issues with vsync at least on 144hz monitors.
+
+	// add quit command
+	command::add("quit", [](command::params&)
+	{
+		// Should be Game::Com_Quit_f();
```
Comment (momo5502) [inline src/client/module/quick_patch.cpp:31]: Yes, it should be...

**Review by momo5502 — CHANGES_REQUESTED** [review]

**Comment by Joelrau** [comment]
I'll be merging quick_patch module into the patch module.

**Pull request closed without merging**